### PR TITLE
Add prefix to redis channel names

### PR DESCRIPTION
**Pull request by BrennanConroy**
@anurse @davidfowl @mikaelm12 @moozzyk 

Fix issue where user and group have the same name

### CODE REVIEW

File: src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs [hunk 1]
```diff
@@ -87,7 +87,7 @@ public override Task InvokeGroupAsync(string groupName, string methodName, objec
                 Arguments = args
             };
 
-            return PublishAsync(typeof(THub).FullName + "." + groupName, message);
+            return PublishAsync(typeof(THub).FullName + ".group." + groupName, message);
```
Comment (mikaelm12) [inline src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs]: Should Group and User be capitalized?

**Comment by BrennanConroy** [comment]
Should I do the same for connectionId channel name? You could technically set the group name or user name to the connectionId

**Comment by BrennanConroy** [comment]
Technically we get connectionId for free because the other 2 have a prefix.

**Comment by mikaelm12** [comment]
Maybe for consistency then

**Pull request merged**